api/v1beta1/manila_types.go

```diff
@@ -143,6 +147,31 @@ func init() {
 	SchemeBuilder.Register(&Manila{}, &ManilaList{})
 }
 
+// DBPurge struct is used to model the parameters exposed to the Manila API CronJob
+type DBPurge struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=30
+	// Age -
+	Age int `json:"age"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="1 0 * * *"
+	//Schedule -
+	Schedule string `json:"schedule"`
+}
+
+// ManilaDebug indicates whether certain stages of Manila deployment should
+// pause in debug mode
+type ManilaDebug struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=false
+	// dbSync enable debug
+	DBSync bool `json:"dbSync,omitempty"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=false
+	// DBPurgeDebug enables debug mode on the DBPurge CronJob
+	DBPurgeDebug bool `json:"debug,omitempty"`
```

Comment:
@ASBishop not sure about your opinion but from a pure CR perspective collecting all the `debug` pieces in a single place helped me to have a better view over the debug area